packages/orm/src/EntityFilter.ts

```diff
@@ -74,6 +74,10 @@ export type ValueFilter<V, N> =
   | { nlike: V | undefined }
   | { ilike: V | undefined }
   | { nilike: V | undefined }
+  | { reg?: V | undefined }
```

Comment:
@chr1sjf0x nice! Could prefer for these to be `regex/iregex/nregex/niregex`, just given the ubiquity of "regex" as a term, but otherwise looks great!

Reply:
Change made 👍 

Ready for merge?

Reply:
Yep, feel free to merge when the build passes